CHANGELOG.md

```diff
@@ -4,6 +4,158 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.7.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.7.0) - 2019-01-01
+* BREAKING
+  * Restrict permission check on repositories and fix some problems (#5314)
+  * How only opened milestones on issues page milestone filter (#5051)
```

Comment:
How ? Did you mean “Have” here ?

Reply:
This changelog is generated with the changelog tool based on the PR titles.

Reply:
sorry removed the S